inventory.py

```diff
@@ -93,57 +99,109 @@ def gather_course_info_for_account(account_id: int, term_id: int) -> Sequence[in
-    udw_courses_string = ','.join([str(udw_course_id) for udw_course_id in udw_course_ids])
+def pull_enrollment_data_from_udw(course_ids) -> pd.DataFrame:
+    courses_string = ','.join([str(course_id) for course_id in course_ids])
     enrollment_query = f'''
-        SELECT e.id AS enrollment_id,
-               e.canvas_id AS enrollment_canvas_id,
-               e.user_id AS enrollment_user_id,
-               e.course_section_id AS enrollment_course_section_id,
-               e.course_id AS enrollment_course_id,
-               e.workflow_state AS enrollment_workflow_state,
+        SELECT e.id AS warehouse_id,
+               e.canvas_id AS canvas_id,
+               e.course_id AS course_id,
+               e.course_section_id AS course_section_id,
+               e.user_id AS user_id,
+               e.workflow_state AS workflow_state,
                r.base_role_type AS role_type
         FROM enrollment_dim e
         JOIN role_dim r
             ON e.role_id=r.id
-        WHERE e.course_id IN ({udw_courses_string});
+        WHERE e.course_id IN ({courses_string});
     '''
 
     logger.info('Making enrollment_dim query')
     enrollment_df = pd.read_sql(enrollment_query, UDW_CONN)
+    logger.debug(enrollment_df.head())
+    return enrollment_df
 
-    enrollment_df.to_csv(os.path.join('data', 'enrollment.csv'), index=False)
-    logger.info('Enrollment data was written to data/enrollment.csv')
 
-    user_ids = enrollment_df['enrollment_user_id'].drop_duplicates().to_list()
+def pull_user_data_from_udw(user_ids: Sequence[int]) -> pd.DataFrame:
     users_string = ','.join([str(user_id) for user_id in user_ids])
-
     user_query = f'''
-        SELECT u.id AS user_id,
-               u.canvas_id AS user_canvas_id,
-               u.name AS user_name,
-               u.workflow_state AS user_workflow_state,
-               p.unique_name AS pseudonym_uniqname
+        SELECT u.id AS warehouse_id,
+               u.canvas_id AS canvas_id,
+               u.name AS name,
+               p.unique_name AS uniqname,
+               u.workflow_state AS workflow_state
         FROM user_dim u
         JOIN pseudonym_dim p
             ON u.id=p.user_id
         WHERE u.id in ({users_string});
     '''
-
     logger.info('Making user_dim query')
     user_df = pd.read_sql(user_query, UDW_CONN)
-    user_df.to_csv(os.path.join('data', 'user.csv'), index=False)
-    logger.info('User data was written to data/user.csv')
+    # Found that the IDs are not necessarily unique, so dropping duplicates
+    user_df = user_df.drop_duplicates(subset=['warehouse_id', 'canvas_id'])
+    logger.debug(user_df.head())
+    return user_df
+
+
+def check_if_valid_user_id(id: int, user_ids) -> bool:
+    if id in user_ids:
+        return True
+    else:
+        return False
+
+
+def run_course_inventory() -> None:
+    start = datetime.now()
+
+    # Gather course data
+    course_df = gather_course_info_for_account(1, ENV['TERM_ID'])
```

Comment:
account id value could be set in the env.json file. It may have different value than the root account.

Reply:
I'll do this!

Reply:
This should be resolved now.

Reply:
See [this commit](https://github.com/tl-its-umich-edu/course-inventory/pull/10/commits/9ab08d5cf186428d81a678fa441cd13ae57146f4).